docs/.vuepress/sidebar.js

```diff
@@ -135,6 +135,13 @@ module.exports = {
       children: [
         [ 'cookbook/windows-development.md', 'Windows development environment' ]
       ]
+    },
+    {
+      title: 'Docker containers',
+      collapsable: false,
+      children: [
+        [ 'cookbook/using-docker.md', 'Setting up an Apostrophe project with Docker' ]
```

Comment:
"Hosting an Apostrophe project" let's be real clear this is about hosting and not dev

Reply:
Changed.